packages/databricks-vscode/src/bundle/migration-template/databricks_template_schema.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Why do we have this file?

Reply:
It's required for all templates. That's how CLI knows what template variables to expect, and in the interactive mode this schema is used to ask questions.